### DISCUSSION TITLE
User for recipe changes

### DISCUSSION THREAD

**Discussion by picposter** [nebhead/PiTender#8] [Q&A] [answered]
Hello,
i installed a fresh raspi pi OS and installed the PiTender software with the auto installer. I went to "raspi-IP:9001 and logged in with the password as i thought this was the way to go. After that i just entered the Raspi Ip and now i can see the WebUi.
Is there a setting to make changes in the recipes and the Admin settings only if logged in? I was instantly able to change settings without logging in on the WebUi?
Or do i have to logout on "raspi-Ip":9001? If yes, how to do that?

Kind regards
Cedric :)

**Comment by nebhead** [comment, marked as answer]
Hey @picposter - Everything is behaving as normal.  Let me try to explain.  

raspi-IP:9001 - This is the ip:port for the Supervisor Daemon.  This program monitors the processes on the Raspberry Pi that we specified (app.py and control.py) during the installation.  It requires a username and password for access to the web UI, but is completely optional.  This ip:port doesn't ever need to be accessed unless you are experiencing issues with the processes running properly.  The username and password have no other usage.  

raspi-IP - As for the normal WebUI, there is no password or username system implemented.  It's intended to be open to anyone who accesses this IP.  There is not a way to restrict access to the recipes currently.  But it could be something that we implement in the future if there is a need.  

I hope this helps!
